Pay/PayAddress.swift

```diff
@@ -0,0 +1,131 @@
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Foundation
+import PassKit
+
+public struct PayPostalAddress {
+    
+    public let city:         String?
+    public let country:      String?
+    public let countryCode:  String?
+    public let province:     String?
+    public let zip:          String?
+    
+    // ----------------------------------
+    //  MARK: - Init -
+    //
+    public init(city: String? = nil,
+         country:     String? = nil,
+         countryCode: String? = nil,
+         province:    String? = nil,
+         zip:         String? = nil) {
+        
+        self.city         = city
+        self.country      = country
+        self.countryCode  = countryCode
+        self.province     = province
+        self.zip          = zip
+    }
+}
+
+public struct PayAddress {
+    
+    public let addressLine1: String?
+    public let addressLine2: String?
+    public let city:         String?
+    public let country:      String?
+    public let countryCode:  String?
+    public let province:     String?
+    public let zip:          String?
+    
+    public let firstName:    String?
+    public let lastName:     String?
+    public let phone:        String?
+    
+    // ----------------------------------
+    //  MARK: - Init -
+    //
+    public init(addressLine1: String? = nil,
+        addressLine2: String? = nil,
+        city:         String? = nil,
+        country:      String? = nil,
+        countryCode:  String? = nil,
+        province:     String? = nil,
+        zip:          String? = nil,
+        
+        firstName:    String? = nil,
+        lastName:     String? = nil,
+        phone:        String? = nil) {
+        
+        self.addressLine1 = addressLine1
+        self.addressLine2 = addressLine2
+        self.city         = city
+        self.country      = country
+        self.countryCode  = countryCode
+        self.province     = province
+        self.zip          = zip
+        
+        self.firstName    = firstName
+        self.lastName     = lastName
+        self.phone        = phone
+    }
+}
+
+// ----------------------------------
+//  MARK: - PassKit -
+//
+internal extension PayPostalAddress {
+    
+    init(with address: CNPostalAddress) {
+        self.init(
+            city:        address.city,
+            country:     address.country,
+            countryCode: address.isoCountryCode,
+            province:    address.state,
+            zip:         address.postalCode
+        )
+    }
+}
+
+internal extension PayAddress {
+    
+    init(with contact: PKContact) {
+        let lines = contact.postalAddress!.street.components(separatedBy: .newlines)
+        self.init(
+            addressLine1: lines.count > 0 ? lines[0] : nil,
+            addressLine2: lines.count > 1 ? lines[1] : nil,
+            city:         contact.postalAddress!.city,
```

Comment:
The `PayAddress` model doesn't make sense without a valid postal address. Agreed, this could be handled better but this functionality is augmented in future PRs.